app/router.js

```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders', {path: '/reminders'});
+  this.route('reminders', {path: '/reminders'}, function() {
```

Comment:
Because the path matches the defined route exactly, we can modify this line to just be `this.route('reminders', function() {....` - Ember assumes the path is the same as the route unless told otherwise.